ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/techpreview/provision/cucushift-installer-rehearse-gcp-ipi-xpn-techpreview-provision-chain.yaml

```diff
@@ -0,0 +1,40 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-techpreview-provision
+  steps: 
+    - ref: upi-conf-gcp-xpn
+    - ref: gcp-provision-private-dns-zone-in-service-project
```

Comment:
I guess this step is not a MUST, suggest to drop it.

Reply:
You know, as there are steps of `gcp-provision-private-dns-zone-in-service-project` & `gcp-provision-private-dns-zone-in-host-project` & `gcp-deprovision-private-dns-zone`. I think all new steps should be tested before the PR getting merged, even if the final workflow won't call these steps. Your suggestion?

Reply:
My suggestion is we can add a dedicated workflow to cover pre-created dns zone cases later.